plugins/optimization-detective/class-od-html-tag-processor.php

```diff
@@ -239,7 +239,7 @@ final class OD_HTML_Tag_Processor extends WP_HTML_Tag_Processor {
 	 *
 	 * @inheritDoc
 	 * @since 0.4.0
-	 * @since n.e.x.t Passing a $query is now allowed. In the 1.0.0 release, this will default to skipping tag closers.
+	 * @since 1.0.0 Passing a $query is now allowed. In the 1.0.0 release, this will default to skipping tag closers.
```

Comment:
This reads a bit weird now, since it's marked `1.0.0` (which makes sense since we don't use betas for `@since`), but also mentions `1.0.0` as the future.

Do we want to maybe put a `TODO` here so that we don't forget adjusting this comment prior to the actual `1.0.0` stable?

Reply:
```suggestion
	 * @since 1.0.0 Passing a $query is now allowed. TODO: In the final non-beta 1.0.0 release, also note that this will default to skipping tag closers.
```